_docs/tasks/security/authn-policy.md

```diff
@@ -88,6 +86,31 @@ NAME          AGE
 enable-mtls   1m
 ```
 
+Add this destination rule to config client side to use mTLS:
+
+```bash
+cat <<EOF | istioctl create -f -
+apiVersion: "networking.istio.io/v1alpha3"
+kind: "DestinationRule"
+metadata:
+  name: "enable-mtls"
+  namespace: "foo"
+spec:
+  host: *.foo.svc.local.cluster
+  trafficPolicy:
+    tls:
+      mode: MUTUAL
```

Comment:
Shouldn't we just go for `ISTIO_MUTUAL` directly? So that user experience is better also we don't have any migration challenge when we have SDS.

Reply:
Can we change this now? The PR is now merged and well tested :)

We can easily ensure the backward compatibility for `ISTIO_MUTUAL` by not for explicit key cert path (which will be replaced by SDS sometime).